templates/projects/task_notification.html

```diff
@@ -5,16 +5,66 @@
 {% from "_formhelpers.html" import render_field %}
 
 {% block projectcontent %}
-<h3>Send email notification on the progress of tasks</h3>
+<h3>Send notification on the progress of tasks</h3>
 <form id="task_notification" method="post" action="{{ url_for('project.task_notification', short_name=project.short_name) }}">
     {{ form.hidden_tag() }}
     <fieldset>
         {{ render_field(form.remaining, placeholder= _('10')) }}
+        <div id="notif-options">
+            <p>{{_('The following options are available:')}}</p>
+            <ul>
+                <li><strong>{{_('Send email notification to project owner(s)')}}</strong>:
+                    {{_('The platform will send email when there are less than or equal to target-remaining tasks left')}}
```

Comment:
Perhaps wording similar to the following below. Note, the Webhook URL field is moved to be just under the remaining tasks textbox. The help text has been moved to the bottom. Also, if we could show the example payload as JSON instead of Python, that could be helpful. Thank you.

**Notify when the number of remaining tasks is less than or equal to:**
[ 4          ]

**Webhook URL**
[ http://www.domain.com/my-web-hook ]
*Optional*

The following notification options are available:

- Email notification: Project owner(s) will receive an email when the number of remaining tasks reaches the threshold. *(Default)*
- Webhook URL: A POST request will be sent to the URL provided with details of the project and number of remaining tasks. See *example payload*.